File: programming languages/28. prova esame/Campo.java
public class Campo {
    private final int codice;
    private final String nome;

    public Campo(int codice, String nome) {
        this.codice = codice;
        this.nome = nome;
    }

    public int getCodice() {
        return codice;
    }

    public String getNome() {
        return nome;
    }
}

File: Fondamenti_programmazione_II/Python/Esami/Gestione_circolo_sportivo/Campo.py
class Campo:
    def __init__(self, codice, campo, larghezza, lunghezza, costo, sport):
        self._codice = codice
        self._campo = campo
        self._larghezza = larghezza
        self._lunghezza = lunghezza
        self._costo = costo
        self._sport = sport
        # self._prenotazioni = 0

    def getSport(self):
        return self._sport

    def getCodice(self):
        return self._codice

    def getCosto(self):
        return self._costo
    
    """
    def addPrenotazioni(self):
        self._prenotazioni += 1

    def incasso(self):
        return self._prenotazioni * self._costo
    """

    def __str__(self):
        return self._sport + "\t" + self._campo + "\t" + str(self._codice) + "\t"